docs/sdks/server-sdks/java.md

```diff
@@ -148,7 +148,6 @@ More details about logging and examples (with Segment, Rudderstack, mParticle, a
 If using the SDK to train and request actions from a contextual multi-armed bandit, you will need to:
 1. Define a bandit assignment logger
 2. Pass bandit actions to request a bandit assignment
-3. (Optional) Log non-bandit actions
```

Comment:
Since we're not going this route, I figured I'd remove this from the documentation and we can remove from the SDK when we port to UFC.